adafruit_ov7670.py

```diff
@@ -584,7 +592,7 @@ def __init__(
             data_pins=data_pins, clock=clock, vsync=vsync, href=href
         )
 
-    def capture(self, buf):
+    def capture(self, buf: Union[bytearray, memoryview]) -> None:
```

Comment:
We have a `WritableBuffer` alias that can be used for `buf` here instead of the Union directly: https://github.com/adafruit/Adafruit_CircuitPython_Typing/blob/e54126d5e9a504775a5d4369beb8b2aae1fe8963/circuitpython_typing/__init__.py#L44

It can be imported like:
```
from circuitpython_typing import WritableBuffer
```
then  used here like:
```
def capture(self, buf: WritableBuffer) - None:
```